themes/rr/rr.theme.sh

```diff
@@ -0,0 +1,29 @@
+#! bash oh-my-bash.module
+
+# rr is a simple one-liner prompt inspired by robbyrussell from ohmyzsh themes.
+#
+# Looks:
+#
+# ➜  anish ~ cd .bash-it/themes/dulcie
+# ➜  anish custom-dulcie git:(master ✓) # with git
+#
+# Configuration. Change these by adding them in your .bash_profile
+
+function _omb_theme_RR_PROMPT() {
+    # configs
+    user_name="${_omb_prompt_white}\u${_omb_prompt_reset_color}"
+    base_directory="${_omb_prompt_bold_blue}\W${_omb_prompt_reset_color}"
+    SCM_THEME_PROMPT_PREFIX="${_omb_prompt_bold_purple}git:(${_omb_prompt_reset_color}"
+    SCM_THEME_PROMPT_SUFFIX="${_omb_prompt_bold_purple})${_omb_prompt_reset_color}"
+    SCM_THEME_PROMPT_CLEAN="${_omb_prompt_bold_green} ✓${_omb_prompt_reset_color}"
+    SCM_THEME_PROMPT_DIRTY="${_omb_prompt_bold_red} ✗${_omb_prompt_reset_color}"
+    arrow="${_omb_prompt_bold_purple}➜${_omb_prompt_reset_color}"
+
+    PS1="${arrow}  ${user_name} ${base_directory} "
+
+    if [[ "$(scm_prompt_info)" ]]; then
+        PS1="${PS1}$(scm_prompt_info) "
+    fi
+}
+
+_omb_util_add_prompt_command _omb_theme_RR_PROMPT
```

Comment:
We are using `_omb_theme_PROMPT_COMMAND` for the prompt setting of all the themes. This is because, when we switch the theme in the same session, we want to overwrite the function for `PROMPT_COMMAND` set by the previous theme. Please also update the name in the function definition.

```suggestion
_omb_util_add_prompt_command _omb_theme_PROMPT_COMMAND
```